src/fprime/fbuild/interaction.py

```diff
@@ -391,7 +391,7 @@ def get_port_input(namespace):
 
     # Fill in blank values with defaults
     for key in values:
-        if values[key] == "":
```

Comment:
I am hesitant here.  If somehow a value is correctly set to the literal `False` the new code will override it, and the old code would keep it.  This only works if `values[keys]` is always a string.  I am not confident that assumption will always hold.

Reply:
Corrected!